app/views/helpers/templates/formHiddenYesNoRadio.scala.html

```diff
@@ -23,6 +24,7 @@
             Seq(
             "true"->Messages("app.common.yes"),
             "false"->Messages("app.common.no")),
+            isHeading = true,
```

Comment:
this is always set to true and doesn't make use of the isHeading parameter passed in to the helper